app/src/androidTest/java/ch/epfl/skysync/database/DatabaseSetup.kt

```diff
@@ -164,6 +179,9 @@ class DatabaseSetup {
             launch { basket2 = basket2.copy(id = basketTable.add(basket2)) },
             launch { vehicle1 = vehicle1.copy(id = vehicleTable.add(vehicle1)) },
             launch { vehicle2 = vehicle2.copy(id = vehicleTable.add(vehicle2)) },
+            launch {
+              messageGroup1 = messageGroup1.copy(id = messageGroupTable.add(messageGroup1))
```

Comment:
Why do you add the messageGroup1 here and not at the end when the the set of user is added ?

Reply:
Because I need the message group id to add the messages, but good catch I need to set the userIds before adding the message group, will fix